docs/_core/theming/creating-theme.story.tsx

```diff
@@ -0,0 +1,83 @@
+import { Source } from '@storybook/components';
+import React from 'react';
+
+export const CreatingTheme = () => (
+  <>
+    <p>Castor components use theme tokens for styling.</p>
+    <p>
+      Most of a time theme tokens will be using base tokens with an alpha
+      variation. But it can also have static values.
+    </p>
+    <p>When creating a new theme, each theme token must be defined.</p>
+    <p>
+      Easiest would be to extend one of Castor themes, for example a
+      &#34;day&#34; theme, and then changing some of theme tokens:
+    </p>
+    <Source
+      language="scss"
+      code={`
+        @use '~@onfido/castor';
+
+        // .castor-theme--custom-a
+        @include castor.theme('custom-a', 'class') {
```

Comment:
We should probably document the requirement for a custom theme to start with `castor-theme--` for the JS helper to function properly.
It also makes it clearer why the `castor.theme()` mixin is needed instead of a simple class rule.

Additionally, we should also explain the `'class' and 'raw'` modifiers for the mixin in that same context, even though we mention them later.